lib/fluent/plugin/out_logdna.rb

```diff
@@ -84,7 +85,8 @@ def send_request(body)
       url = "/logs/ingest?hostname=#{@host}&mac=#{@mac}&ip=#{@ip}&now=#{now}&tags=#{@tags}"
       @ingester.headers('apikey' => @api_key,
                         'content-type' => 'application/json')
-               .post(url, json: body)
+                .timeout(connect: @request_timeout_seconds, write: request_timeout_seconds, read: request_timeout_seconds)
```

Comment:
I think you should use `@request_timeout_seconds` for all instead of `request_timeout_seconds` here, no?

Reply:
Good catch! Actually not sure how the UTs passed since `request_timeout_seconds` isn't declared....